simulator/modules/core/perf_sim.cpp

```diff
@@ -13,6 +13,12 @@
 namespace config {
     static const AliasedValue<std::string> units_to_log = { "l", "logs", "nothing", "print logs for modules"};
     static const Switch topology_dump = { "tdump", "module topology dump into topology.json" };
+    // command line flag argument for enable/disable prefetch
+    static const AliasedSwitch prefetch = { "p", "prefetch", "cache prefetching" };
+    // command line argument for fetchahead distance size
+    static const Value<uint32> fetchahead_distance = { "fetchahead-size", 32, "fetchahead distance size"};
+    // command line argument for prefetch type
+    static const Value<std::string> prefetch_method = {"prefetch-method", "wrong_path", "Type of prefetching method"};
```

Comment:
Can use "prefetch_method" variable to enable prefetches as well by having a third option "no-prefetch"?
Also, please specify that is a Instruction prefetch.
We are likely to have data prefetches once there is data cache.

Reply:
Is it worth deleting the "prefetch" variable in this case?

Reply:
Yes